datavault-assembly/src/datavault-home/bin/start-worker.sh

```diff
@@ -1 +1,13 @@
-# This is a wee shell script to strat the work, only I haven't written it yet....
\ No newline at end of file
+#!/bin/bash
+
+if [ -z "$DATAVAULT_HOME" ]; then
+    echo "DATAVAULT_HOME is not set"
+    exit 1
+fi
+
+cd ${DATAVAULT_HOME}/lib
```

Comment:
Is it better not to cd into the lib directory (so the user stays in the same directory), and instead to add 'lib/' to the jar in the -cp classpath?